app/assets/javascripts/i18n.js

```diff
@@ -497,6 +510,63 @@
     return message.replace(/_#\$#_/g, "$");
   };
 
+  // Not part of the public API
+  I18n.splitInterpolate = function(message, matches, options) {
+    var value
+    , placeholder
+    , name
+    , regex;
+
+    var buffer = [message];
+
+    while (matches.length) {
+      placeholder = matches.shift();
+      name = placeholder.replace(this.placeholder, "$1");
+
+      if (this.isSet(options[name])) {
+        value = options[name];
+      } else if (name in options) {
```

Comment:
Again, good comment, but the same thing here. That's how it was in the original implementation and I wouldn't want to include too many refactorings here.